pkgs/os-specific/linux/kernel/cachyos-kernels.nix

```diff
@@ -0,0 +1,311 @@
+{
+  lib,
+  fetchFromGitHub,
+  pkgs,
+  stdenv,
+  buildLinux,
+  ...
+}@args:
+
+let
+  cachyPatches = fetchFromGitHub {
+    owner = "CachyOS";
+    repo = "kernel-patches";
+    rev = "04fb9178158a7714aeb908c7a5310c4e0d6ea4b6";
```

Comment:
Add a comment here like, "when updating this package, make sure to account for important config changes in https://github.com/CachyOS/linux-cachyos/blob/master/linux-cachyos/config as well"